asio/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.14...3.23)
+
+include(cmake/prelude.cmake)
+
+project(asio
+        VERSION 1.22.1
+        DESCRIPTION "Asio C++ Library"
+        HOMEPAGE_URL "http://think-async.com/Asio/"
+        LANGUAGES CXX
+)
+
+include(cmake/project-is-top-level.cmake)
+include(cmake/variables.cmake)
+
+if(PROJECT_IS_TOP_LEVEL)
+  include(GNUInstallDirs) # for CMAKE_INSTALL_INCLUDEDIR
```

Comment:
This `include` is needed in this context!

Reply:
This is not needed here, the code that deals with installing is in the `install-rules.cmake` file.